lib/request.js

```diff
@@ -38,9 +49,44 @@ module.exports = (options) => {
             bodyStream.on('end', () => {
                 response.body = Buffer.concat(chunks).toString('utf8');
                 hasRequestEnded = true;
+                if (options.timing) {
+                    timings.end = Date.now() - startTime;
+                    response.timingStart = startTime;
```

Comment:
`timings.start` seems inconsistent to me because `timings` object has relative times, but this `timingStart` is an absolute time. But maybe we don't need `timingStart` itself. I cannot think of its use case for now. I'll remove it.